pkg/controller/container-runtime-config/helpers.go

```diff
@@ -439,7 +439,7 @@ func validateUserContainerRuntimeConfig(cfg *mcfgv1.ContainerRuntimeConfig) erro
 
 	ctrcfg := cfg.Spec.ContainerRuntimeConfig
 	if ctrcfg.PidsLimit != nil && *ctrcfg.PidsLimit != 0 && *ctrcfg.PidsLimit < minPidsLimit {
-		return fmt.Errorf("invalid PidsLimit %q", *ctrcfg.PidsLimit)
+		return fmt.Errorf("invalid PidsLimit %v", *ctrcfg.PidsLimit)
```

Comment:
Absolutely non-blocking: Maybe use `%d` to be explicit about the intended format, and let the linters help with inconsistencies? OTOH it isn’t really likely for the type of `PidsLimit` to change.